workflows/prognostic_c48_run/runtime/derived_state.py

```diff
@@ -31,6 +31,9 @@ def __getitem__(self, key: str) -> xr.DataArray:
         if key == "time":
             time = self._getter.get_state(["time"])["time"]
             return xr.DataArray(time, name="time")
+        elif key == "initialization_time":
+            initialization_time = self._getter.get_state([key])[key]
+            return round_time(cftime.DatetimeJulian(*initialization_time.timetuple()))
```

Comment:
I think this should basically be the same as the `key=='time'` clause. It's a bit confusing to be doing rounding (to the nearest second?) and also this function is supposed to return a `xr.DataArray` but it looks like this case would return a `cftime.DatetimeJulian`.

Reply:
Sure, changed it to returning a data array. Our setup has multiple ways of getting the time in the wrapper (this was the other way), which is potentially confusing.